pkg/v2/service.go

```diff
@@ -287,6 +287,15 @@ func (s *service) Create(ctx context.Context, r *taskAPI.CreateTaskRequest) (_ *
 			Options: m.Options,
 		})
 	}
+
+	rootfs := ""
+	if len(mounts) > 0 {
```

Comment:
Why is this check needed? `rootfs` can now be an empty string, while it would  always be set before.

Reply:
I suppose that this is to support the case when their are no Rootfs mounts passed to the CreateTaskRequest. Not sure how one could create a valid container without a rootfs though.

For normal containers maybe it would be to support using the host mount namespace? Not sure that would apply in gVisor's case.

Reply:
I don't think we need `len(mounts) > 0`, will remove it.

Reply:
Done